docs/production/dbos-cloud/workflow-management.md

```diff
@@ -29,7 +29,7 @@ If the workflow is enqueued, cancelling removes it from the queue.
 
 #### Resuming Workflows
 
-You can resume any `ENQUEUED`, `CANCELLED` `PENDING`, or `RETRIES_EXCEEDED` workflow.
+You can resume any `ENQUEUED`, `CANCELLED`, or `RETRIES_EXCEEDED` workflow.
```

Comment:
The sentence below also needs to be fixed -- `Resuming a workflow sets its status to `PENDING` and resumes its execution from its last completed step.` should be changed to `Resuming a workflow resumes its execution from its last completed step.`